src/components/LessonCard.tsx

```diff
@@ -33,9 +34,13 @@ const Title: React.FC<TitleProps> = (props) => {
   );
 };
 
-const MenuKebab = () => {
+const MenuKebab = (CardId: any) => {
```

Comment:
type "any" is bad practice

Reply:
Yes, it is better to declare proper type for it

Reply:
done